### PR TITLE
Adds code languages to troubleshooting docs

### PR DESCRIPTION
**Pull request by mkandler**

### CODE REVIEW

**Comment by mkandler** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #335** <a href="https://app.graphite.dev/github/pr/run-house/runhouse/335?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/run-house/runhouse/335?utm_source=stack-comment).

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ you may perform the following manual setup steps.
 
 1. Install Runhouse on your local box with:
 
-   .. code::
+   .. code:: ipython3
```
Comment (carolineechen) [inline docs/troubleshooting.rst:16]: I think this can be shell, same as the pip install below

Comment (mkandler) [reply docs/troubleshooting.rst:16]: oh yup! thanks for catching that

File: docs/troubleshooting.rst [hunk 2]
```diff
@@ -22,21 +22,21 @@ you may perform the following manual setup steps.
...
+   .. code:: shell
 
         pip install runhouse  # latest
         # or
         pip install git+https://github.com/run-house/runhouse.git  # main branch
 
 4. Start Ray on the remote box:
 
-   .. code::
+   .. code:: ipython3
```
Comment (carolineechen) [inline docs/troubleshooting.rst:39]: oops this one too should be shell

**Review by carolineechen — APPROVED** [review]

**Pull request merged**